### ISSUE TITLE
Still cannot run swiftlint on circleci

### ISSUE THREAD

**Issue by brennantaylor** [realm/SwiftLint#593]
Still getting dyld: Library not loaded

I saw an issue was closed earlier but it doesn't look resolved.

In another project, swiftgen, it looks like a workaround was done in homebrew by not pouring from bottle if there is no `/Applications/Xcode.app`. https://github.com/Homebrew/homebrew/pull/45008

This would at least get it running for now.

**Comment by norio-nomura** [comment]
Thanks for reporting.
We have PR #555 that will resolve this issue.

**Comment by brennantaylor** [comment]
Oh I'm sorry I should have looked through the open pull requests more closely.

Thanks for the update!

**Comment by jpsim** [comment]
@brennantaylor how's this looking now that #555's been merged?

**Comment by brennantaylor** [comment]
@jpsim I can't seem to compile HEAD with brew at the moment. Since there is no release either I can't easily install on circle. I can test as soon as the next release comes out.

brew install --HEAD swiftlint says `** CLEAN SUCCEEDED **` but then exits uncleanly.

**Comment by norio-nomura** [comment]
@brennantaylor That issue has been reproduced on my local mac.
It seems that is same as https://github.com/Carthage/Carthage/issues/1243

**Comment by norio-nomura** [comment]
[SwiftLint 0.10.0](https://github.com/realm/SwiftLint/releases/tag/0.10.0) has been released and is expected to run on CircleCI with `brew install swiftlint`.

**Comment by jpsim** [comment]
If someone confirms that it works, we'll close this issue, so just let us know!

**Comment by mikemee** [comment]
This worked on CircleCI just now:
 * SwiftLint installed ok with `brew install swiftlint` (_see below_)
 * the previous warning about SwiftLint not being installed went away
 * the previous warning about SwiftLint not being installed went away
 * the build succeeded
 * an intentional fatal SwiftLint error in my code made the build fail
 * warnings did not fail (nor did they display anything in the logs -- I'll be making more warnings fail soon!) 

Fwiw, my `circle.yml` file was (Carthage references left there to avoid unintentionally editing away some crucial detail, but see [here](https://robots.thoughtbot.com/caching-carthage-con-circleci) if you're interested):

```yaml
general:
  branches:
    only:
      - master

machine:
  xcode:
    version: 7.3.1

dependencies:
  override:
    - brew install swiftlint
    - bin/bootstrap-if-needed
  cache_directories:
    - "Carthage"

```

Awesome awesome tool. Thanks Realm for making it available and supporting it.  (And, together with the video transcriptions, this made me find realm and now I'm using it my apps...)

**Comment by jpsim** [comment]
Great to hear this works now 😄
